src/interfaces/coral_web/src/components/Settings/SettingsTab.tsx

```diff
@@ -36,16 +33,6 @@ export const SettingsTab: React.FC = () => {
     });
   };
 
-  if (isLangchainModeOn) {
-    return (
-      <div className="flex items-center justify-center px-5">
-        <Text styleAs="p-lg" className="select-none text-center text-volcanic-100">
-          {STRINGS.langchainMultihopSettingsDisabledDescription}
```

Comment:
probably we can remove this variable from the `STRINGS` constant

Reply:
Sounds good